shell/src/main/java/org/apache/accumulo/shell/commands/ConfigCommand.java

```diff
@@ -117,7 +118,8 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
       }
     } else if (cl.hasOption(setOpt.getOpt())) {
       // set property on table
-      String property = cl.getOptionValue(setOpt.getOpt()), value = null;
+      String property = cl.getOptionValue(setOpt.getOpt());
+      String value;
```

Comment:
Thank you for that!
We should sweep through our code in a separate PR and fix all those multi-assignments (except the ones in try-with-resources or for loop initialization, of course).

Reply:
By fix do you mean replace with multiple assignments, like in this case?

Reply:
> By fix do you mean replace with multiple assignments, like in this case?

Yes, where it makes sense to do it, anyway.

Reply:
Created #3689 to do as follow-on.